create_tables.sql

```diff
@@ -42,13 +42,6 @@ EXCEPTION
  WHEN duplicate_object THEN null;
 END $$;
 --> statement-breakpoint
-CREATE TABLE IF NOT EXISTS "zoning_district" (
```

Comment:
Let's keep the creation and import of zoning_district here because it's not a source table, but one we want to create and load transformed data into. I wanted to remove pluto creation and import because your load_sources.sql handles that.

Reply:
fixed